server/build/webpack.js

```diff
@@ -33,17 +33,13 @@ function babelConfig (dir, {isServer, dev}) {
   }
```

Comment:
Same as below about `babel.createConfigItem` for the hot loader listing above this.